feature-libs/asm/package.json

```diff
@@ -40,4 +42,4 @@
     "access": "public"
   },
   "schematics": "./schematics/collection.json"
-}
+}
```

Comment:
This might fail with the prettier formatting or linting. Files should end with a new line.